### ISSUE TITLE
Results Leader Board Not Showing

### ISSUE THREAD

**Issue by Hansxsourse** [codalab/codalab-competitions#3101]
Hi, I just find that the leader board of this [competition](https://codalab.lisn.upsaclay.fr/competitions/721#results) is not showing. I tried change browser and change device, it still not showing, at the same time other competition's leader board can be seen at my end. Can anyone help me to solve it out?

**Comment by Didayolo** [comment]
Hi,

I think you need to be logged in and to join the competition to be able to see the leaderboard.

You can request to participate here:
https://codalab.lisn.upsaclay.fr/competitions/721#participate

**Comment by Hansxsourse** [comment]
> Hi,
> 
> I think you need to be logged in and to join the competition to be able to see the leaderboard.
> 
> You can request to participate here: https://codalab.lisn.upsaclay.fr/competitions/721#participate

The issue is happened while I've logged in and also paticipated in.

**Comment by Didayolo** [comment]
Indeed I can't see the results neither. It's strange since the organizer of the competition seems to see it well.

**Comment by boazarad** [comment]
I'm the competition organizer, and seem to have the same problem.
The leaderboard appears as seen below - submission numbers and high-scores are visible, but we do not see a table with the top results and users.

I've received feedback from some participants that are seeing the same partial display.

![image](https://user-images.githubusercontent.com/571510/154059711-d1dd1319-c56b-4a3e-80db-1557c868199d.png)

**Comment by Didayolo** [comment]
Thank you for jumping in.

Did it work in the past?

**Comment by boazarad** [comment]
Thanks for the quick reply - the table has been missing since the challenge started.
I've tried changing some settings to fix this (such as checking and unchecking "If submission beats old score, put submission on leaderboard") but nothing seems to have made a difference.

This competition was created based on a YAML template of the NTIRE2020 competition on the old codalab site, perhaps that is part of the issue?

**Comment by mayorx** [comment]
> I'm Adding the competition YAML and example output of the scoring program in hopes that this might shed some light on the issue, any help would be greatly appreciated as our competitors are having a lot of trouble progressing without a leaderboard:
> 
> [competition.yaml.txt](https://github.com/codalab/codalab-competitions/files/8176620/competition.yaml.txt)
> 
> [scores.txt](https://github.com/codalab/codalab-competitions/files/8176632/scores.txt)

Since this issue isn't going to be resolved, why not recreate a new codalab competition and let's all submit in a new place?
Another workaround is to update the results daily to a url (like Google Sheets), if you as the organizer can see the results, just let us know what the url is. 
Also, something like this [post](https://codalab.lisn.upsaclay.fr/forums/721/206/) says can barely be used as a way for us to know approximate rankings. Why has it been more than a month and still no progress?
@boazarad

**Comment by boazarad** [comment]
To clarify - the primary issue is the missing leaderboard.

A user suggested a workaround: if the result graph would display more significant digits, users would have a better idea of their relative placement vs. each days best score.

Setting `numeric format=4` does not affect the graph display, and is not a good workaround for the missing leaderboard.

**Comment by Didayolo** [comment]
OK I get it now.

**Comment by Didayolo** [comment]
@Tthomas63 @dtuantran 
Any thought on this bug?

**Comment by samsara-ku** [comment]
Hi! I'm having trouble with same issue. 

I have made and deleted many competitions because not familiar with this platform, and I think a important thing on this issue is...

There is probably **NO** correlation between the content of `configuration.yaml` and this situation. I have made many competitons with same yaml files, but sometimes I could see my result on the leaderboard but most of times couldn't see it.

So, is there anyone who can solve this issue quickly? I'm looking forward solving this issue...
